packages/dx-chart-core/src/plugins/series/computeds.js

```diff
@@ -165,3 +165,10 @@ export const seriesData = (series = [], seriesProps) => {
 };
 
 export const checkZeroStart = (fromZero, axisName, pathType) => ({ ...fromZero, [axisName]: fromZero[axisName] || (pathType === 'area' || pathType === 'bar') });
+
+export const getItemsCallback = (pluginName) => {
+  if (pluginName === 'PieSeries') {
```

Comment:
Usually we extract string values to `const` variables

Reply:
Ok